doc/user/zebra.rst

```diff
@@ -1058,6 +1058,12 @@ zebra Terminal Mode Commands
    nexthop groups that do have an afi. [type] allows you to filter those
    only coming from a specific NHG type (protocol).
 
+.. index:: show <ip|ipv6> zebra route dump [<vrf> VRFNAME]
```

Comment:
Instead of creating a new command, how about augmenting `show ip route`?

Here is a suggestion:
`show ip route details`

Reply:
How strongly do you feel about this? This PR has been open for a year, I'd like to take care of it and move on.